airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/job_tracker/JobTracker.java

```diff
@@ -268,11 +271,16 @@ public void trackSync(Job job, JobState jobState) {
     }
   }
 
-  private void track(UUID workspaceId, Map<String, Object> metadata) {
+  private void track(final UUID workspaceId, final Map<String, Object> metadata)
+      throws JsonValidationException, ConfigNotFoundException, IOException {
+    final Map<String, Object> standardTrackingMetadata = ImmutableMap.of(
+        "workspace_id", workspaceId,
+        "workspace_name", configRepository.getStandardWorkspace(workspaceId, true).getName());
+
```

Comment:
How come this is done in `JobTracker..java` as opposed to in the `TrackingClient.java` (ie `SegmentTrackingClient`)  since other classes than  `JobTracker` are also calling the track providing `workspaceId`?

Reply:
We can merge as it is and change this later on